src/Message.ts

```diff
@@ -3,14 +3,28 @@ export enum MessageType {
     Poll = "poll"
 }
 
-export interface VideoMessage {
+export interface Friend {
+    id: string;
+    muted: boolean;
+    title: string;
+}
+
+export interface LocalVideoMessage {
     type: MessageType.Video;
     currentTime: number;
     paused: boolean;
 }
 
-export interface PollMessage {
+export interface LocalPollMessage {
     type: MessageType.Poll;
 }
 
-export type Message = VideoMessage | PollMessage;
+export interface RemoteMessageExtensions {
+    friends: Friend[];
```

Comment:
I think sending friends and trying to ensure connections based on the friends added to the message, we might get into some pretty interesting race conditions while users are connecting and disconnecting while people are interacting with the video

Reply:
Yea, I think this again comes to splitting this work of peer-connnection-circle from the whole `isSharedControl` and `muted` features.

It's valuable work, but I don't want to have PRs that aren't mergable b/c they snowball.